ui/src/components/Common/Demo/apiInterceptor.ts

```diff
@@ -1,10 +1,11 @@
 import axios from 'axios'
-import { BASE_REST_URL, BASE_REST_AUTH } from '../../../../devConfig'
+//import { BASE_REST_URL, BASE_REST_AUTH } from '../../../../devConfig'
```

Comment:
If axiosAuth is no longer used anywhere, should be able to delete this file and devConfig.

Reply:
For now we can't remove these files as these are helpfull to develop without VM. Will remove these files after all feather components development.